src/components/BarChart/Chart.tsx

```diff
@@ -134,78 +138,73 @@ const BarChart = ({
   }, [])
 
   // pan responder is responsible for the slide interaction
-  const panResponder = useMemo(
-    () =>
-      PanResponder.create({
-        onMoveShouldSetPanResponder: (e, gestureState) => {
-          return Math.abs(gestureState.dy) < 10
-        },
-        onPanResponderMove: (evt) => {
-          const dataIndex = findDataIndex(evt.nativeEvent.locationX)
-          setFocusedBar(data[dataIndex])
-        },
-        onPanResponderRelease: () => {
-          setFocusedBar(null)
-        },
-      }),
+  const onPanEvent = useCallback(
+    (event: { nativeEvent: PanGestureHandlerEventPayload }) => {
+      const dataIndex = findDataIndex(event.nativeEvent.x)
+      setFocusedBar(data[dataIndex])
+    },
     [data, findDataIndex],
   )
 
   return (
-    <Animated.View
-      style={{
-        backgroundColor: 'rgba(255,0,0,0)',
-        width,
-        height,
-      }}
-      onTouchStart={handleTouchStart}
-      onTouchEnd={handleTouchEnd}
-      // eslint-disable-next-line react/jsx-props-no-spreading
-      {...panResponder.panHandlers}
+    <PanGestureHandler
+      activeOffsetX={[-5, 5]} // only activate the gesture if x moves 5 pixels
+      onGestureEvent={onPanEvent}
+      onEnded={handleTouchEnd}
     >
-      <Svg height="100%" width="100%">
-        {data.map((v, i) => (
-          <React.Fragment key={`frag-${v.id}`}>
-            <Rect
-              x={barWidth * (2 * i)}
-              y={maxUpBarHeight - barHeight(v?.up)}
-              rx={barWidth / 2}
-              width={barWidth}
-              height={barHeight(v?.up)}
-              fill={upColor || greenBright}
-              opacity={!focusedBar || focusedBar?.id === v.id ? 1 : 0.4}
-            />
-
-            {hasDownBars && (
+      <Animated.View
+        style={{
```

Comment:
Could probably cleanup this inline style.

Reply:
Moved it out and use memo